src/components/scripts/InfoBar.js

```diff
@@ -0,0 +1,16 @@
+//Function used to update info bar, displays score, county, town, long, lat
+function InfoBar(props) {
+  return (
+    <div className="InfoBar">
+      <div>Score:{props.score} </div>
+      <div>County: {props.county} </div>
+      <div>
+        Town: {props.city} {props.village} {props.hamlet} {props.town}{" "}
+      </div>
+      <div>Latitude: {props.latitude} </div>
+      <div>Longitude: {props.longitude} </div>
+    </div>
+  );
+}
+
+export default InfoBar;
```

Comment:
Something that stood out to me in the file was that it seems to be the only file that you did not de structure `props` in. I'm curious as to why other larger components with many props being passed in used de structured props - but this smaller component where destructing would be easiest does not. 

It may be useful to keep things consistent across files just for the sake of being consistent.